tests/baselines/summary_2023-08-05T15:01:52.604Z.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
These summaries are auto-generated and persist over multiple runs, moreover, I was thinking of not to setting `results` folder to a path inside Django project, instead it can go to a location like `/tmp` or `~/.acheker/results`.

I added the results folder here just to showcase everything.